docs/MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+## Maintaining VTEX Toolbelt
+
+This document is for people working on VTEX as Toolbelt maintainers.
+
+### Forcing a new version
+
+Toolbelt has a way of enforcing users to use a minimum version choosed by us. This enforcement is done on the client side (can be disabled by an environment variable, check [EnvVariables](../src/lib/constants/EnvVariables.ts)) and on `vtex.builder-hub` - users using an older toolbelt version will not be allowed to link or publish. This feature is useful for enforcing users to update to the latest version, which can be important when we launch a new version with a major bug fix for example.
```

Comment:
I understood why it is interesting for us, but why is it interesting for users? Maybe: 

"This feature is useful for enforcing users to update to the latest version, which can be important when we launch a new version with a major bug fix for example, making then know quickly we are improving their experiences, saving time to read documentations about new releases." ?